src/content/ui-components/components/blockquote-button.mdx

```diff
@@ -1,11 +1,11 @@
 ---
 title: Add a button to toggle block-level nodes
 meta:
-  title: Node button | Tiptap UI Component
-  description: Toggle through block-level nodes like code blocks and blockquotes with the node button. More in the docs.
+  title: Blockquote Button | Tiptap UI Component
+  description: Toggle through block-level nodes like blockquotes with the Blockquote Button. More in the docs.
```

Comment:
Add a button that toggles through blockquotes in your Tiptap editor. More in the docs!